### PR TITLE
[Merged by Bors] - chore: backports for leanprover/lean4#4814 (part 21)

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Unital.lean [hunk 1]
```diff
@@ -322,14 +322,16 @@ lemma cfc_cases (P : A → Prop) (a : A) (f : R → R) (h₀ : P 0)
     · rwa [cfc_apply_of_not_continuousOn _ h]
 
 variable (R) in
-lemma cfc_id : cfc (id : R → R) a = a :=
+lemma cfc_id (ha : p a := by cfc_tac) : cfc (id : R → R) a = a :=
```
Comment (sgouezel) [inline Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Unital.lean:325]: Maybe remove `ha` from Line 279, if it's never included automatically? Same for lines 280 and 281?

**Review by sgouezel — COMMENTED** [review]
bors d+

File: Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Unital.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Unital.lean:325]: These are fairly often included automatically, when they appear in the statements.

**Comment by semorrison** [comment]
(The "delegation" label bot is dying because of API rate limits, so I added it myself.)

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**